src/main/java/com/fasterxml/jackson/databind/annotation/JsonPOJOBuilder.java

```diff
@@ -85,6 +85,6 @@ public Value(String buildMethodName, String withPrefix)
 	        this.withPrefix = withPrefix;
 	    }
 
-	    public static Value empty() { return EMPTY; }
```

Comment:
This is backwards incompatible change, unless change was done for 2.9? If so, shouldn't `empty` remain as deprecated and new one added. Also please add `@since 2.9` for new `defaults()` method